src/core/Akka/Util/TypeExtensions.cs

```diff
@@ -37,6 +38,16 @@ public static bool Implements(this Type type, Type moreGeneralType)
         {
             return moreGeneralType.IsAssignableFrom(type);
         }
+
+        /// <summary>
+        /// Utility to be used by implementors to create a manifest from the type.
+        /// The manifest is used to look up the type on deserialization.
+        /// </summary>
+        /// <returns>Returns the type qualified name including namespace and assembly, but not assembly version.</returns>
+        public static string TypeQualifiedName(this Type type)
```

Comment:
We have the same method in Persistence. But we need it everywhere, not only in serializers